egs/wsj/s5/steps/nnet3/tdnn/train.sh

```diff
@@ -119,6 +123,15 @@ if [ $# != 4 ]; then
   echo "                                                   # Frame indices used for each splice layer."
   echo "                                                   # Format : layer<hidden_layer_index>/<frame_indices>....layer<hidden_layer>/<frame_indices> "
   echo "                                                   # (note: we splice processed, typically 40-dimensional frames"
+  echo "  --cnn-layer <string|cnn-layer0/filt_x_dim=3 filt_y_dim=8 filt_x_step=1 filt_y_step=1 num_filters=256 pool_x_size=1 pool_y_size=3 pool_z_size=1 pool_x_step=1 pool_y_step=3 pool_z_step=1> "
```

Comment:
I'd rather not allow the shell script to accept multiple versions of the
same parameter.  This will be hard to implement and will break the normal
shell-level interface.
Dan


On Thu, Feb 25, 2016 at 11:50 AM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/wsj/s5/steps/nnet3/tdnn/train.sh
> <https://github.com/kaldi-asr/kaldi/pull/526#discussion_r54125545>:
>
> > @@ -119,6 +123,15 @@ if [ $# != 4 ]; then
> >    echo "                                                   # Frame indices used for each splice layer."
> >    echo "                                                   # Format : layer<hidden_layer_index>/<frame_indices>....layer<hidden_layer>/<frame_indices> "
> >    echo "                                                   # (note: we splice processed, typically 40-dimensional frames"
> > +  echo "  --cnn-layer <string|cnn-layer0/filt_x_dim=3 filt_y_dim=8 filt_x_step=1 filt_y_step=1 num_filters=256 pool_x_size=1 pool_y_size=3 pool_z_size=1 pool_x_step=1 pool_y_step=3 pool_z_step=1> "
>
> does shell script allow for multiple specification of --cnn-layer
> parameters ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/526/files#r54125545>.
>